docs/src/appendix/forge-library.md

```diff
@@ -8,6 +8,7 @@
 * [`parse_txt`](forge-library/parse_txt.md) - parses plain text file content and tries to deserialize it into the specified type
 * [`read_json`](forge-library/read_json.md) - reads and parses json file content into an array of felts
 * [`parse_json`](forge-library/parse_json.md) - parses json file content and tries to deserialize it into the specified type
+* [`env`](forge-library/env.md) - module containing functions for interacting with the system environment
```

Comment:
I would just describe `var` function here. I get the intention, but should stick to current convention until
#719 and then refactor the docs alltogether